guides/deployment/releases.md

```diff
@@ -172,10 +172,10 @@ Elixir releases work well with container technologies, such as Docker. The idea
 Here is an example Docker file to run at the root of your application covering all of the steps above:
 
 ```docker
-FROM elixir:1.9.0-alpine AS build
+FROM hexpm/elixir:1.11.2-erlang-23.1.2-alpine-3.12.1 as build
 
 # install build dependencies
-RUN apk add --no-cache build-base npm git python
+RUN apk add --no-cache build-base npm git python3
```

Comment:
Just out of curiosity, what depends on python?

Reply:
Node.js uses gyp (via [node-gyp](https://github.com/nodejs/node-gyp)) for compiling native addons.

Reply:
Thank you 🙏